src/components/molecules/AssetStatus.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import Status from '../atoms/Status'
+import Tooltip from '../atoms/Tooltip'
+import styles from './AssetStatus.module.css'
+
+export default function AssetStatus({
+  isDisable
+}: {
+  isDisable?: boolean
+}): ReactElement {
+  const [status, setStatus] = useState('')
+
+  useEffect(() => {
+    isDisable === true && setStatus('Disabled')
+  }, [isDisable])
+
+  return isDisable ? (
+    <div className={styles.wrapper}>
+      <Status className={styles.status} state="error" />
+      <span className={styles.text}>{status}</span>
+      <Tooltip
+        className={styles.info}
+        content="This dataset might be empty/offline, consume at your own risk."
```

Comment:
this message does not fit the proposed state. "Disabling" is a dedicated user action whereas this message refers to "Unavailable" which might be untrue which this message reflects ("might be")